pkg/core/native/native_test/neo_test.go

```diff
@@ -118,6 +118,47 @@ func TestNEO_CandidateEvents(t *testing.T) {
 	require.Equal(t, 0, len(aer.Events))
 }
 
+func TestNEO_VoteForNull(t *testing.T) {
```

Comment:
OK, you need to register and vote for new committee to properly update voters reward. This test becomes very similar to `TestNEO_Vote`, thus let's get back to the initial plan and extend `TestNEO_Vote` instead of creating a new test. In `TestNEO_Vote` you already have new committee elected, so the task is in the end of `TestNEO_Vote` to take one voter that has voted for the new committee member, check its account state, then perform unvoting andthen check its account state one more time.

Reply:
The only thing is that you have to re-register candidate 0 back and wait for committee update one more time, because test ends with rolling back to standby validators:
https://github.com/nspcc-dev/neo-go/blob/1679b162b8d33695f63c56a472db2c98b36e26e6/pkg/core/native/native_test/neo_test.go#L264-L265